MainModule/Client/Core/Anti.lua

```diff
@@ -78,7 +81,7 @@ return function(Vargs, GetEnv)
 
 	local Detected = function(action, info, nocrash)
 		if NetworkClient and action ~= "_" then
-			pcall(Send, "D".."e".."t".."e".."c".."t".."e".."d", action, info)
+			pcall(Send, "D".."e".."t".."e".."c".."t".."e".."d", hyperionEnabled and "log" or action, tostring(info)..(hyperionEnabled and " - Hyperion is enabled, not giving punishment" or isXbox and " - On Xbox" or isMobile and " - On mobile" or ""))
```

Comment:
is this correct? this doesnt seem like it would work

Reply:
probably left over from when this PR used to not give punishments, but now it just shows this message, except I guess editing the message to clarify that was just too much work.

Reply:
Oh this was accidentally left here. The point was only to help with debugging instead of changing the punishment mechanism.